README.md

```diff
@@ -1,4 +1,4 @@
-# Fest ![Logo](/res/icons/hicolor/24x24/apps/fest.svg)
+# Fest <img src="/res/icons/hicolor/24x24/apps/fest.svg" height="24" width="24">
```

Comment:
What's the point of this change? Having this as MD is preferable to having it as HTML.

Reply:
Compat with browsers, they don't always respect size of SVG and require size provided to them